emscripten.py

```diff
@@ -2623,8 +2644,36 @@ def create_receiving_wasm(exports):
 ''' % {'mangled': asmjs_mangle(e), 'e': e, 'assertions': runtime_assertions})
 
   if not shared.Settings.SWAPPABLE_ASM_MODULE:
-    for e in exports:
-      receiving.append('var %(mangled)s = Module["%(mangled)s"] = asm["%(e)s"];' % {'mangled': asmjs_mangle(e), 'e': e})
+    if shared.Settings.DECLARE_ASM_MODULE_EXPORTS:
+      if shared.Settings.WASM and shared.Settings.MINIMAL_RUNTIME:
```

Comment:
why does wasm2js output differ from wasm output here? in general in the wasm backend the js/wasm paths are much more similar than in fastcomp (wasm2js literally polyfills the WebAssembly APIs, and everything should be identical except for that), so I was hoping we would not have differences here.

Reply:
I don't know yet. I have not yet tried against wasm2js. I just put this like this to follow the fastcomp route, and did not go too far making modifications to keep the amount to review small for each PR.